web/src/engine/websites/StarboundScans.ts

```diff
@@ -19,4 +38,42 @@ export default class extends DecoratableMangaScraper {
         return icon;
     }
 
+    public override ValidateMangaURL(url: string): boolean {
+        return new RegExpSafe(`^${this.URI.origin}/webtoon/[^/]+$`).test(url);
```

Comment:
Website is tagged with Manga, but URL indicates that there are no mangas, is this by intention or is an URL pattern missing?

Reply:
I pay no attention to website url like that, websites tends to use series, manga, manhwa, manga, whatever and have multiple type of content. In that case, turns out you are right.